src/katgpucbf/xbgpu/beamform.py

```diff
@@ -105,6 +111,8 @@ class Beamform(accel.Operation):
         Number of frequency channels
     n_spectra_per_frame
         Number of samples in time axis for each frame (fine time dimension)
+    seed, sequence_first, sequence_step
+        See :class:`.RandomStateBuilder`.
```

Comment:
Don't mean to be that guy
* This doesn't render as an actual hyperlink to `src/katgpucbf/curand_helpers.py::RandomStateBuilders`.

Admittedly, I don't have a suggestion to fix this
* It would be nice if this was updated to link to the correct location, but
* I wouldn't hold this PR up over that.

Reply:
You should totally be that guy. I'm that guy all the time, and it improves quality.

In this case it's because I failed to do the checklist properly when I added the curand_helpers.py module, so it's not linked into the documentation. The good news is I already fixed that in another PR in the meantime, and when I merge main into this branch, it fixes the rendering.